Cargo.toml

```diff
@@ -39,8 +39,11 @@ name = "ui"
 required-features = ["render"]
 
 [dependencies]
-bevy = { version = "0.13", default-features = false, features = ["bevy_asset"] }
+bevy = { version = "0.14.0-rc.2", default-features = false, features = [
```

Comment:
```suggestion
bevy = { version = "0.14.0-rc.3", default-features = false, features = [
```

Reply:
This is not necessarily needed, a `cargo update` will also get you onto `rc.3`, but this won't hurt.